### PR TITLE
raidboss: convert most sb triggers to output strings

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

File: ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js [hunk 1]
```diff
@@ -222,17 +222,20 @@
...
         // If for some reason there is no partner, we get a vulnerability or bleed and are sad.
         if (!data[partner])
           return;
-        return {
-          en: 'Stack with ' + data.ShortName(data[partner]),
-          de: 'Sammeln mit ' + data.ShortName(data[partner]),
-          fr: 'Packez-vous avec ' + data.ShortName(data[partner]),
-          cn: '靠近' + data.ShortName(data[partner]) + '集合',
-        };
+        return output.text({ player: data.ShortName(data[partner]) });
```
Comment (panicstevenson) [inline ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js:230]: nit: `partner`?

File: ui/raidboss/data/04-sb/raid/o2s.js [hunk 2]
```diff
@@ -98,71 +94,76 @@
...
-            en: 'DPS up, T/H down',
-            de: 'DDs hoch, T/H runter',
-            ja: 'DPS浮上 タンクヒラ降下',
-            cn: 'DPS升起，T奶下降',
-            ko: '딜러 공중부양, 탱힐 강하',
-          };
-        }
+      infoText: function(data, _, output) {
+        if (!data.role.startsWith('dps'))
+          return output.dpsUpThDown();
```
Comment (panicstevenson) [inline ui/raidboss/data/04-sb/raid/o2s.js:142]: needless nit: `dpsUpTanksHealersDown()`?

File: ui/raidboss/data/04-sb/trial/suzaku-ex.js [hunk 3]
```diff
@@ -13,27 +13,30 @@
...
-          return {
-            en: 'Buster on ' + data.ShortName(matches.target),
-            de: 'Tankbuster auf ' + data.ShortName(matches.target),
-            fr: 'Tank buster sur ' + data.ShortName(matches.target),
-            ja: data.ShortName(matches.target) + 'にタンクバスター',
-            cn: '死刑 点' + data.ShortName(matches.target),
-            ko: '"' + data.ShortName(matches.target) + '" 탱버',
-          };
-        }
+      alertText: function(data, matches, output) {
```
Comment (panicstevenson) [inline ui/raidboss/data/04-sb/trial/suzaku-ex.js:16]: Response maybe?

**Review by panicstevenson — APPROVED** [review]

File: ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js (hunk 1 above)
Comment (quisquous) [reply ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js:230]: I think I'm going to stick with `${player}` for any player names, for consistency.

File: ui/raidboss/data/04-sb/raid/o2s.js (hunk 2 above)
Comment (quisquous) [reply ui/raidboss/data/04-sb/raid/o2s.js:142]: Thanks, done!

File: ui/raidboss/data/04-sb/trial/suzaku-ex.js (hunk 3 above)
Comment (quisquous) [reply ui/raidboss/data/04-sb/trial/suzaku-ex.js]: Done.

**Pull request merged**